### ISSUE TITLE
Moved Jackett to other computer. How to uninstall from RPi 4

### ISSUE THREAD

**Issue by Sjonnie2018** [Jackett/Jackett#13130]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using http Client: HttpWebClient2
Using FlareSolverr: Disabled
Using Proxy: Disabled
App config.log directory: folder path
Running in Docker: No
Jackett variant: CoreLinuxArm32
OS version: Unix 3.10.108.0
Environment version: 6.0.3 (/volume1/@appstore/jackett/share/)
```

### Description

I have moved Jackett from my RPi4 to a Synology NAS. All working as expected.

I would like to completely remove Jackett from the RPi. The RPi has Sonarr installed but I would expect that not to be a problem (Sonarr is now looking at the Syno for indexers).

How do I remove Jackett from my RPi?

TIA

Greetz,

Sjonnie

### Logged Error Messages

```text
N/A
```

### Screenshots

N/A

**Comment by ilike2burnthing** [comment]
I would assume the same way you remove any application, e.g. https://forums.raspberrypi.com/viewtopic.php?t=205803 (I've never used a Raspberry Pi)

**Comment by Sjonnie2018** [comment]
Hi,

Thanks for your reply! I (of course) tried "sudo apt remove jackett" and sudo dpkg -l jackett" The last to check if dpkg could actually find the program. But alas! Both options did not work.

If anyone has ever removed Jackett from a Pi I would be very interested to learn how to do that :)

**Comment by ilike2burnthing** [comment]
Does it show as installed in the `Add/Remove Software` application?

**Comment by Sjonnie2018** [comment]
Hi,

Nope! It does not. Which is not surprising as the software is installed using this tutorial:

https://github.com/Jackett/Jackett#installation-on-linux-armv6-or-below)Installation on Linux (ARMv6 or below

Screenshot of add/remove programs:

![image](https://user-images.githubusercontent.com/41323431/162577774-2f6e1fe5-529a-461d-84f9-cc86a1a7c645.png)

**Comment by ilike2burnthing** [comment]
Oh ffs, of course 🤦 

Are you running it manually or did you install it as a service?

If the former, delete the files and uninstall any dependencies you no longer need.

If the latter, [stop and remove the service](https://superuser.com/questions/513159/how-to-remove-systemd-services), then delete the files and uninstall any dependencies you no longer need.

**Comment by Sjonnie2018** [comment]
Hi,

Thanks for your continued support. Much appreciated!!

It does run as a service. I checked with this command:

`sudo systemctl status jackett.service
● jackett.service - Jackett Daemon
   Loaded: loaded (/etc/systemd/system/jackett.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2022-04-09 16:27:55 CEST; 26min ago
 Main PID: 5573 (sh)
    Tasks: 13 (limit: 4915)
   CGroup: /system.slice/jackett.service
           ├─5573 /bin/sh /opt/jackett/jackett_launcher.sh
           └─5576 /opt/jackett/jackett --NoRestart

Apr 09 16:28:06 RPi4 jackett[5573]: 04-09 16:28:06 Info Adding aggregate indexer ('all' indexer) ...
Apr 09 16:28:06 RPi4 jackett[5573]: 04-09 16:28:06 Info Adding filter indexer ('type:public' indexer) ...
Apr 09 16:28:06 RPi4 jackett[5573]: 04-09 16:28:06 Info Adding filter indexer ('type:private' indexer) ...
Apr 09 16:28:06 RPi4 jackett[5573]: 04-09 16:28:06 Info Adding filter indexer ('type:semi-public' indexer) ...
Apr 09 16:28:06 RPi4 jackett[5573]: 04-09 16:28:06 Info Deleting JackettUpdate temp files from /tmp/JackettUpdate-v0.20.818-637851184601520092
Apr 09 16:28:06 RPi4 jackett[5573]: 04-09 16:28:06 Info Jackett startup finished in 11.442 s
Apr 09 16:28:06 RPi4 jackett[5573]: Hosting environment: Production
Apr 09 16:28:06 RPi4 jackett[5573]: Content root path: /opt/jackett/Content
Apr 09 16:28:06 RPi4 jackett[5573]: Now listening on: http://[::]:9117
Apr 09 16:28:06 RPi4 jackett[5573]: Application started. Press Ctrl+C to shut down.
`

I the stopped the service and disabled the service:
`sudo systemctl stop jackett.service
pi@RPi4:~ $ sudo systemctl disable jackett.service
Removed /etc/systemd/system/multi-user.target.wants/jackett.service.`

I then removed the directory /opt/jackett and rebooted the device (just to be sure :-) )

The Pi is running as expected and Jackett has been properly removed.

Again: thanks for your support!
